### PR TITLE
commands/license and commands/ststem: add some e2e tests (batch 4)

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
Adds e2e tests for:
- `removeLicenseCmdF`
- `uploadLicenseCmdF`
- `getBusyCmdF`
- `setBusyCmdF`
- `clearBusyCmdF`

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-31421
- https://mattermost.atlassian.net/browse/MM-31422
- https://mattermost.atlassian.net/browse/MM-31425
- https://mattermost.atlassian.net/browse/MM-31426
- https://mattermost.atlassian.net/browse/MM-31427

### CODE REVIEW

File: commands/system_e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlE2ETestSuite) TestGetBusyCmd() {
```
Comment (mgdelacroix) [inline commands/system_e2e_test.go:17]: These tests are duplicated on https://github.com/mattermost/mmctl/pull/339, right?

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1: Beware of the duplicates!!

File: commands/system_e2e_test.go (hunk 1 above)
Comment (isacikgoz) [reply commands/system_e2e_test.go:17]: Haha, yeah. Good catch, I probably and accidentally indexed this file to this branch 🤦‍♂️

**Review by mkraft — APPROVED** [review]

**Review by lindy65 — APPROVED** [review]
Thanks @isacikgoz - these commands will be tested after merge 👍 

Please add Zephyr (TM4J) test keys to s.Run lines:

image

image

**Pull request merged**